.travis.yml

```diff
@@ -126,10 +126,16 @@ jobs:
 
     - os: linux # deploy pip package
       dist: bionic
+      compiler: gcc
       env:
         - TASK="deployment"
-      install: echo "Skipping install stage!"
+        - ROOTBIN="v6.18.04.Linux-ubuntu18-x86_64-gcc7.4"
       before_install: echo "Skipping before_install stage!"
+      install: # install dependencies
+        - python3 -m pip install scikit-build
+        - wget https://root.cern.ch/download/root_${ROOTBIN}.tar.gz
+        - tar xpvfz root_*.tar.gz > /dev/null 2>&1
+        - source root/bin/thisroot.sh
```

Comment:
do you really need this? I checked before and we only upload the source dist, so building should not be necessary right?

Reply:
Yes I also saw that, neither did we install `scikit-build` previously, right? But then I don't understand why we receive [this error message](https://travis-ci.com/github/ComPWA/pycompwa/jobs/321134950#L998) now...

Reply:
Okay removed that stuff again https://github.com/ComPWA/pycompwa/pull/87/commits/2798f46c358337002644f42da64586e398ab119c

Reply:
Hope that renewing the PyPI key the actual fix
https://github.com/ComPWA/pycompwa/pull/87/commits/88391d1a56648d912a9a2fb7f524b7f830c35b57